mw-webapp/src/component/wayCollectionCard/WayCollectionCard.tsx

```diff
@@ -55,12 +115,42 @@ export const WayCollectionCard = (props: WayCollectionProps) => {
       value={
         <VerticalContainer className={styles.wayCollectionCardContainer}>
           <VerticalContainer className={clsx(styles.mainInfo, props.isActive && styles.active)}>
-            <Title
-              level={HeadingLevel.h3}
-              text={props.collectionTitle}
-              className={styles.title}
-              placeholder=""
-            />
+            {props.isEditable ?
+              <HorizontalContainer className={styles.collectionTitleBlock}>
+                {title}
+                <Dropdown
+                  className={styles.dropdown}
+                  trigger={(
+                    <Tooltip
+                      content={LanguageService.user.collections.collectionActionsTooltip[props.language]}
+                      position={PositionTooltip.LEFT}
+                      isShownOnClick={false}
```

Comment:
Answering to the Viktor's question, I suggested to revert my changes related to the tooltip (it will be working as any item with tooltip in the app as before), and create for me an issue for 2 problems with them. 
I need confirmation from you :)

Reply:
Done